docs/how-to/ownership.mdx

```diff
@@ -0,0 +1,106 @@
+   ![Add Attributes](/ui-videos/how-to/ownership/1.png)
+4. In the resource editor, under `ABAC Options`, click on `Add Attribute`.
+   ![Resource Editor](/ui-videos/how-to/ownership/2.png)
+5. Name the attribute, and set it as a string. Click save.
+   ![Add Attribute](/ui-videos/how-to/ownership/3.png)
+6. Go to the `ABAC Rules` tab, and enable ABAC Options.
+7. Create the following ABAC Resource Set:
+   ![Resource Set](/ui-videos/how-to/ownership/4.png)
+8. Go back to the policy editor screen. You should now be able to see a new file-ownership resource set you can manage the permissions for.
+   ![Policies](/ui-videos/how-to/ownership/5.png)
+
+The ABAC condition will compare the file_ownership attribute (On the resource) is equal to the `user.key` of the user trying to perform the action.
+
+## ReBAC
+
+ReBAC allows us to implement ownership in a more effective way by leveraging existing application-level relationships and creating simple policies based on hierarchies and groupings.
+
+By creating Parent-Child hierarchies, ReBAC allows us to nest resources under other resources.
+Such a relationship allows us to derive roles based on the relationship between two resources:
+
+:::tipExample
+![ReBAC Example](/ui-videos/how-to/ownership/6.png)
+:::
+
+The ReBAC policy for this graph will look like this:
+
+A user who is assigned the role `Folder#Owner` will also be assigned the `File#Owner` when the `Folder` instance is the `Parent` of a `File` instance.
+
+In simpler words, if a file resides inside a folder (That’s the Parent-Child relationship), and a user is assigned the owner of that folder, they will also be the owner of the files.
+
+### Pros & Cons
+
+While providing the obvious benefit of managing hierarchical ownership scenarios, ReBAC might be a little more complex to model.
+
+### Setting up ReBAC based Ownership with Permit
+
+:::tipExample Policy
+For the purpose of this example, let's use the same one from before:
+`Bob` can `Read`, `Upadte`, and `Delete` every file he is an `Owner` of.  
+:::
+
+To setup an ABAC based ownership policy in Permit's UI -
+
+1. Login to your Permit account at app.permit.io
+2. For the purpose of this tutorial, we setup a resource called `File`, a resource called `Folder`, and a role called `User` in advance.
+3. In the `Policy` section, go to the `Resources` tab. Click on `Add Attributes`.
+   ![Resource Editor](/ui-videos/how-to/ownership/7.png)
+4. Under "ReBAC Options", add a resource role named "owner" to both the `File` and `Folder` resources. In the `Folder` resource, add the following relation:
+   ![ReBAC Options](/ui-videos/how-to/ownership/8.png)
+5. Going back to the `Policy Editor` screen, you will be able to see the `file#owner` role and the `folder#owner` role.
+   As `file` is set as parent of `folder`, all permissions set on a `folder` will automatically propogate to `files` within those folders as well.
+   ![Policy Editor](/ui-videos/how-to/ownership/9.png)
```

Comment:
Add a step where you add Bob as owner to a specific instance

Reply:
Done.